### ISSUE TITLE
Fix insecure content issue on payment popup

### ISSUE THREAD

**Issue by erommel** [Arsenal21/stripe-payments#159]
Any content loaded via `http` into payment popup produces `insecure content` browser error and this prevents payments from working normally in Live mode (also in test mode for some addons like APM as it requires https even for test mode payments).

One of the examples of the content that currently could be loaded via `http` is product thumbnail. Some logic should be made to force thier URLs to be `https` in Live mode and in test mode if APM addon is enabled.

We cannot just force all of them to be `https` as this would make self-hosted or SSL-less staging\dev\testing sites (like the one I'm using myself on localhost) not display product thumbnail on payment popup in test mode.

In conclusion, the code should do following:

Check if Live mode is enabled OR APM addon is enabled. If it is, change ALL `http` to `https` on payment popup. If not - perhaps, show some warning that Live mode should be all https if any http resource is found(?)

**Comment by Arsenal21** [comment]
Showing warning in the popup is the best course of action in my opinion.

We can put a warning that says something like the following:
----
Stripe live checkout requires your site to use HTTPS URLs. We have detected that this page contains some content that is not loaded using HTTPS URL. Please check the following and then try again:

1) Make sure the product images are using HTTPS URLs
2) Anything else
3) Anything else that should be checked

**Comment by erommel** [comment]
I have added some checks in code to make sure things are using `https`. However, we definitely don't want to have a built-in HTTPS Everywhere-like plugin ;-) So warning is the best option.
